src/Helper/SplitStreet.php

```diff
@@ -77,6 +77,8 @@ public static function splitStreet(string $fullStreet, string $local, string $de
             foreach (self::NUMBER_SUFFIX_ABBREVIATION as $from => $to) {
                 $fullStreet = preg_replace("/(\d.*-?)[\s]$from/", '$1' . $to, $fullStreet);
             }
+
+            $fullStreet = trim(preg_replace('/(\r\n)|\n|\r/', ' ', $fullStreet));
```

Comment:
I see the same thing happening on line 92. You can take that away. If you took away that line, I will accept your pull-request.

Reply:
Duplicate code removed.